docs/Commands.md

```diff
@@ -0,0 +1,38 @@
+# Commands
+
+Commands in theos are implemented as Makefile targets, and executed with
+
+`make <commands>`
+
+These can often be combined to accomplish multiple things in one go. Theos includes a variety of commands for everything from packaging to installation.
+
+### Packaging
+
+`make package` - Build a package
+
+`make clean package` - Clean the build directory and rebuild the project. If you've updated a header without touching any source files, you may need to use the clean command to see the update in your build.
+
+`make do` will build and install a package, and is a shortcut for `make package install`
+
+`make package FINALPACKAGE=1` will optimise assets (runs [pincrush](https://github.com/DHowett/pincrush) on PNG images, and converts plists to binary format) and generate a package with a “clean” version (ie, no build number). Recommended when building a package you’re about to release.
```

Comment:
I think we say it runs pincrush somewhere else in the wiki but that’s outdated (removed it because there’s no arm64/x86_64 builds of pincrush and it’s a bit pointless to re-encode pngs given modern Apple hardware is incredibly fast).